server/src/main/java/server/haengdong/presentation/response/ActionsResponse.java

```diff
@@ -8,7 +8,7 @@ public record ActionsResponse(
 ) {
     public static ActionsResponse of(List<ActionAppResponse> actions) {
         return new ActionsResponse(actions.stream()
-                .map(ActionResponse2::of)
-                .toList());
+                                           .map(ActionResponse2::of)
+                                           .toList());
```

Comment:
```suggestion
return new ActionsResponse(actions.stream()
                .map(ActionResponse2::of)
                .toList())
```
또는
```suggestion
return new ActionsResponse(
                actions.stream()
                .map(ActionResponse2::of)
                .toList()
        );
```
c: 인덴트가 뭔가 이상해요